modules/op-deleting-the-pipelines-component-and-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/uninstalling-pipelines.adoc
+
+[id='op-deleting-the-pipelines-component-and-custom-resources_{context}']
+= Deleting the {pipelines-title} components and custom resources
+
+Delete the custom resources created by default during installation of the {pipelines-title} Operator.
+
+[discrete]
+== Procedure
+. In the *Administrator* perspective of the Web Console, navigate to *Operators->Installed Operators*.
+
+. In the *Project* drop-down select the *OpenShift-Operators* project.
+
+. Click on the *OpenShift Pipelines Operator* to see the overview of the Operator.
+
+. Click on the *OpenShift Pipelines Config* tab to see the *Configs* page.
```

Comment:
I see this tab in OCP 4.3 but not in 4.4

Reply:
@ppitonak @praveen4g0 @nikhil-thomas - I tried doing an install on a 4.4 cluster, and did not see the config screen. Is this a bug in 4.4, or some functionality has changed, or I am missing something here.

Reply:
@andrewballantyne do you know why it changed?

Reply:
@ppitonak Interesting, cost of making everything internal I think. There was a slack conversation with @nikhil-thomas a while back where we discussed internal APIs and what we should expose as part of the Operator. It seems this PR https://github.com/openshift/console/pull/3877 set out to fix some issues exposing internal configurations.

I'm not sure what the config page had, but I suspect you could search for the resource and delete it that way if needed.

Reply:
This entire procedure is now replaced with a new procedure shared by Pavol.